.github/workflows/build-book.yaml

```diff
@@ -61,6 +72,9 @@ jobs:
         run: mamba env update -n ${{ inputs.environment_name }} -f ${{ inputs.environment_file }}
 
       - name: Build the book
+        with:
+          ARM_USERNAME: ${{ secrets.ARM_USERNAME }}
+          ARM_PASSWORD: ${{ secrets.ARM_USERNAME }}
```

Comment:
Should this be `${{ secrets.ARM_PASSWORD }}`?

Reply:
Yes... good catch!